server/utils/cloudinary.js

```diff
@@ -2,9 +2,9 @@ const cloudinary = require('cloudinary');
 require('dotenv').config();
 
 cloudinary.config({
-  cloud_name: process.env.CLOUD_NAME,
-  api_key: process.env.CLOUD_API_KEY,
-  api_secret: process.env.CLOUD_API_SECRET,
+  cloud_name: process.env.CLOUDINARY_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_API_SECRET,
```

Comment:
Let's make sure to reflect these changes upon the `sample.env` file as well (located [here](https://github.com/hatchways/team-reindeer/blob/feature-integrate-profile-picture-uploading-with-upload-route-backend/server/sample.env)) - `sample.env` is supposed to communicate with other contributors what environment variables need to be set up so everyone can configure and run the application without a problem.